File: code/c/control-flow/test-if.c
/* Program: test-if.c */

#include <stdio.h>

int main()
{
    int num, num1;
    
    printf("Enter a number: ");
    scanf("%d", &num);
    
    if (num != 2) 
        printf("Num is not 2!\n");
    
    printf("Enter another number: ");
    scanf("%d", &num1);
    
    if (num1 == 2 && num != 2)
        printf("You got the hint... num1 is 2!");
    
    
    if (num > num1)
        printf("The first number you entered was the larger.");
    else
        printf("The first number you entered was not larger.");
    
    return 0;
}

File: code/pascal/control-flow/TestIf.pas
program TestIf;

procedure Main();
var
    num, num1: Integer;
begin
    Write('Enter a number: ');
    ReadLn(num);
    
    if num <> 2 then
        WriteLn('Num is not 2!');
    
    Write('Enter another number: ');
    ReadLn(num1);
    
    if (num1 = 2) and (num <> 2) then
        WriteLn('You got the hint... num1 is 2!');
    
    if num > num1 then
        WriteLn('The first number you entered was the larger.')
    else
        WriteLn('The first number you entered was not larger.');
end;

begin
    Main();
end.